docs/docs/migration-guide.mdx

```diff
@@ -10,6 +10,32 @@ description: |
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+## Rasa 2.3.3 to Rasa 2.3.4
+
+:::caution
+This is a release **breaking backwards compatibility of machine learning models**.
```

Comment:
@dakshvar22 Couldn't we have made that one non-breaking? Seem less than ideal to do a model breaking change  in a micro release.

Reply:
@wochinge It wasn't really possible to do in a non-breaking way because that would have meant leaving the bug around for all `2.3.x` versions. Here's the [slack](https://rasa-hq.slack.com/archives/C36SS4N8M/p1614079756041500) thread about it.